schema/tests/data/chapter/negative/logo-small-invalid.yaml

```diff
@@ -0,0 +1,19 @@
+country: India
+leaders:
+  - github: leader-1-github
+    name: Leader 1 Name
+  - github: leader-2-github
+    name: Leader 2 Name
+    slack: leader-2-slack
+name: OWASP Chapter
+social_media:
+  - platform: youtube
+    url: https://www.youtube.com/channel/UC1234567890abc
+tags:
+  - chapter-tag-1
+  - chapter-tag-2
+  - chapter-tag-3
+logo:
+  - small: https://xyz
+    medium: https://example.com/mediumLogo
+    large: https://example.com/largeLogo
```

Comment:
Please reorder test yaml files too -- it's much easier to read schema/tests when it has some expected order.